tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -236,13 +236,14 @@ def _get_input(self, xp, in_type, dtype):
 @testing.parameterize(*testing.product({
     'func': [
         lambda x, y: x + y,
+        lambda x, y: x - y,
     ],
     'type_l': ['poly1d', 'python_scalar'],
     'type_r': ['poly1d', 'ndarray', 'python_scalar', 'numpy_scalar'],
 }))
 class TestPoly1dArithmetic(Poly1dTestBase):
 
-    @testing.for_all_dtypes()
+    @testing.for_all_dtypes(no_bool=True)
```

Comment:
Don't skip bool polyadd test.

Reply:
bool tests also need to be skipped for polymul that's why I separated bool and non bool tests